lib/checkout.rb

```diff
@@ -0,0 +1,25 @@
+require 'dotenv/load'
+require 'twilio-ruby'
+
+class Checkout

```

Comment:
Perhaps there is a better name for this class, as this class only deals with sending a text message? A later class might deal with actual checkout, which would involve payment.